torcx/troubleshooting.md

```diff
@@ -0,0 +1,56 @@
+## Troubleshooting
+
+Torcx generator runs early in the boot, when other system facilities are not yet set up and available for use. In case of errors, troubleshooting and debugging can be performed following the suggestions described here.
+
+### Checking for failures
+
+In case of errors, Torcx stops before sealing the new system state. This means that in order to check for correct execution, it is sufficient to verify that the metadata file exists:
+
+```
+$ test -f /run/metadata/torcx || echo 'torcx failed'
+```
+
+On failures, the metadata seal file won't exist thus printing the "torcx failed" output. This can be also verified at boot-time using the `torcx.target` unit:
```

Comment:
On failures, the metadata seal fill will not exist, and `torcx failed` will be printed. Verify failure at boot time using the `torcx.target` unit.